recipes/torch-pme/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "torch_pme" %}
+{% set version = "0.1.0" %}
+
+package:
+  # use name with a hyphen to be consistent with PyPI name
+  name: torch-pme
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7d31801b44a19a1633ecc4f2c484811342940b074dcd01f30dd97ff5a4e69278
+
+build:
+  # conda-forge's pytorch is not yet available on Windows
+  skip: true  # [win or py<39]
```

It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows. 

```suggestion
  noarch: python
```